src/routes/plans/[id]/+page.svelte

```diff
@@ -170,4 +176,43 @@
     on:changeColumnSizes={changeColumnSizes}
     on:changeRowSizes={changeRowSizes}
   />
+  <Console>
+    <svelte:fragment slot="console-tabs">
+      <div class="console-tabs">
+        <div>
+          <ConsoleTab numberOfErrors={$allErrors?.length}>All</ConsoleTab>
+        </div>
+        <div class="separator">|</div>
+        <div class="grouped-error-tabs">
+          <ConsoleTab numberOfErrors={$simulationDatasetErrors?.length}><PlayIcon /></ConsoleTab>
```

Comment:
Can we add a simple tooltip for each of these tabs that displays the title? (i.e. simulation errors, scheduling errors)